### DISCUSSION TITLE
eww bar always in front of all other windows even I set stacking "bg" on Archlinux X11 bspwm

### DISCUSSION THREAD

**Discussion by Gandharikos** [elkowar/eww#971] [Q&A]
![image](https://github.com/elkowar/eww/assets/12139734/a13a6cdf-97eb-4f14-8ae3-3857335d6dc4)
bar still in front of my window when I full screen
my environment is Arch linux X11 bspwm
This is my bar code
```
(defwindow bar
  :geometry
    (geometry
      :x "0"
      :y "5px"
      :width "99.5%"
      :height "25px"
      :anchor "top center"
    )
  :stacking "bg"
  :reserve (struts :distance "35px" :side "top")
  :windowtype "dock"
  :wm-ignore false
  (bar)
)

```

**Comment by NikitaGolovaniuk** [comment]
Same system and same problem. It works well untill i reload eww and then i can remove it from fg lol

**Comment by NikitaGolovaniuk** [comment]
MAN I FOUND A SOLUTION, put - xdo lower -N Eww at the end of ur bspwmrc

**Comment by NikitaGolovaniuk** [comment]
But my solution makes an another trouble - panel stop react on hovers and clicks. I cant get whats wrong why if i just use panel everything is ok but if i reload bsppwm cfg or type  smth or reload eww - panel wents on top of everything

**Reply by genga898** [reply]
You might want to add `:exclusive true` this tells the wm to make space for it, it is a wayland only option if  I am not wrong

**Comment by JustYousefSameh** [comment]
Any updates about this?

**Comment by recursivevoidptr** [comment]
i use bspwm + eww bar and i changed the windowtype to "desktop" and it works fine.

**Reply by ankkax** [reply]
Tried it now I can't click the widgets.
